frontend/ui/widget/progresswidget.lua

```diff
@@ -44,7 +44,7 @@ local ProgressWidget = Widget:new{
     bordercolor = Blitbuffer.COLOR_BLACK,
     bgcolor = Blitbuffer.COLOR_WHITE,
     nlcolor = Blitbuffer.COLOR_LIGHT_GRAY,
-    rectcolor = Blitbuffer.COLOR_DIM_GRAY,
+    rectdim = 2/3,
```

Comment:
I'd rather you use something more clearly in the eInk palette ;) (That'd be `0x44`, I think).

Reply:
That's not a color, that's the dim factor, wheee. Never mind me :).

Reply:
But this is not a color, but an "alpha" factor.